src/components/Dropdown/V2/Dropdown.css.js

```diff
@@ -125,74 +125,86 @@ export const ActionContentUI = styled('div')`
   max-width: 100%;
 `
 
-export const ItemUI = styled('div')`
-  ${baseStyles};
-  background-color: ${rgba(getColor('grey.400'), 0)};
-  border-radius: 0 !important;
-  color: ${getColor('charcoal.400')};
-  cursor: pointer;
-  display: block;
-  outline: none;
-  position: static;
-  transition: background-color 0.1s ease;
-  user-select: none;
+export const makeItemUI = element => {
+  const ItemUI = styled(element)`
+    ${baseStyles};
+    background-color: ${rgba(getColor('grey.400'), 0)};
+    border-radius: 0 !important;
+    color: ${getColor('charcoal.400')};
+    cursor: pointer;
+    display: block;
+    outline: none;
+    position: static;
+    text-decoration: none;
+    transition: background-color 0.1s ease;
+    user-select: none;
+
+    &.is-option {
+      padding: 8px 16px;
+    }
 
-  &.is-option {
-    padding: 8px 16px;
-  }
+    &.is-active {
+      > ${ActionUI}, &.is-option {
+        font-weight: 500;
+      }
+    }
 
-  &.is-active {
-    > ${ActionUI}, &.is-option {
-      font-weight: 500;
+    &.c-SelectionClearerItem + .c-DropdownV2Item {
+      padding-top: 12px;
     }
-  }
 
-  &.c-SelectionClearerItem + .c-DropdownV2Item {
-    padding-top: 12px;
-  }
+    &:focus {
+      ${MenuUI} {
+        color: initial;
+      }
+    }
 
-  &:focus {
-    ${MenuUI} {
-      color: initial;
+    &:hover {
+      text-decoration: none;
     }
-  }
 
-  &:last-child {
-    border: none;
-  }
+    &:last-child {
+      border: none;
+    }
 
-  ${WrapperUI} {
-    position: absolute;
-    padding-left: 20px;
-    padding-right: 20px;
-    margin-left: -20px;
-    margin-right: -20px;
-  }
+    ${WrapperUI} {
+      position: absolute;
+      padding-left: 20px;
+      padding-right: 20px;
+      margin-left: -20px;
+      margin-right: -20px;
+    }
 
-  &.is-focused {
-    > ${ActionUI}, &.is-option {
-      background-color: ${rgba(getColor('grey.300'), 1)};
-      color: ${getColor('link.base')};
+    &.is-focused {
+      > ${ActionUI}, &.is-option {
+        background-color: ${rgba(getColor('grey.300'), 1)};
+        color: ${getColor('link.base')};
+      }
     }
-  }
-  &.is-focused:hover,
-  &.is-open {
-    > ${WrapperUI} {
-      visibility: visible;
-      pointer-events: auto;
+    &.is-focused:hover,
+    &.is-open {
+      > ${WrapperUI} {
+        visibility: visible;
+        pointer-events: auto;
+      }
     }
-  }
 
-  &.is-disabled {
-    cursor: initial;
-    pointer-events: none;
-    opacity: 0.4;
+    &.is-disabled {
+      cursor: initial;
+      pointer-events: none;
+      opacity: 0.4;
+    }
+  `
+
+  ItemUI.defaultProps = {
+    [SELECTORS.itemAttribute]: true,
   }
-`
-ItemUI.defaultProps = {
-  [SELECTORS.itemAttribute]: true,
+
+  return ItemUI
 }
 
+export const ItemUI = makeItemUI('div')
```

Comment:
In case anyone was using this outside of our library, I have continued to export `ItemUI` as a `div` which is backwards compatible.